spec/jobs/bulkrax/delete_work_job_spec.rb

```diff
@@ -34,6 +35,7 @@ module Bulkrax
       it 'increments :deleted_records' do
         expect(importer_run).to receive(:increment!).with(:deleted_records)
         expect(importer_run).to receive(:decrement!).with(:enqueued_records)
+        entry.save
```

Comment:
? seems like maybe there aren't any specs for this?

Reply:
not really. I can hash out the specs for the delete job a bit more next week if you want.